antarest/matrixstore/repository.py

```diff
@@ -336,3 +338,10 @@ def delete(self, matrix_hash: str) -> None:
         # Abandoned lock files are deleted here to maintain consistent behavior.
         lock_file = matrix_path.with_suffix(".tsv.lock")
         lock_file.unlink(missing_ok=True)
+
+    def get_matrix_disk_usage(self, matrix_hash: str) -> int:
+        for internal_format in InternalMatrixFormat:
```

Comment:
we'd better extract a small method `_find_matrix_path` that will be used also by other functions that already perfom the same lookup